src/integrationTest/java/uk/gov/hmcts/reform/draftstore/controllers/LivenessHealthCheckTest.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.draftstore.controllers;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource("/database.properties")
+public class LivenessHealthCheckTest {
+
+    @LocalServerPort
+    private int port;
+
+    @Test
+    public void should_return_UP_for_liveness_check() {
```

Comment:
no point adding an integration check for the starter imo, its tested there and doesn't need a separate integration test

Reply:
sure ... will remove.